### DISCUSSION TITLE
Noob question command line longitude

### DISCUSSION THREAD

**Discussion by HeinBloed32** [Stellarium/stellarium#2094] [General]
As a newbie to stellarium i am confused about using the command line option --longitude. 
When i enter stellarium --longitude=-0.5, stellarium opens in position E 0° 30' instead of W 0°30'.
Entering --longitude=0.5 (instead of -0.5) does give the same position. 
Same error when using format -0d30'

What  is my error ?

Thanks

**Comment by gzotti** [comment]
It seems nobody has used this for years now, and the sign is chopped off. You can try 359.5 for now, but we should fix that.

**Reply by HeinBloed32** [reply]
i guess it is not the sign, it is a calculation error: stellarium longitude = parameter value subtracted by 1.  Entering --longitude-23.5 results in W 22°30'.

**Reply by gzotti** [reply]
Which version did you check? I know where to look, and we changed this bit recently. But it's not the easiest code.

**Comment by HeinBloed32** [comment]
1.2.2 Windows 64 Desktop



Georg Zotti ***@***.***> schrieb am So. 12. Dez. 2021 um
23:49:

> Which version did you check? I know where to look, and we changed this bit
> recently. But it's not the easiest code.
>

**Comment by gzotti** [comment]
Presumably 0.21.2 then? The changes of the last weeks did then at least not change this behaviour, but I admit something is wrong here.

**Comment by HeinBloed32** [comment]
Yes, download number is different from Version number (Impressum)


Georg Zotti ***@***.***> schrieb am Mo. 13. Dez. 2021 um
09:53:

**Comment by HeinBloed32** [comment]
Hi Georg,

i recognized some changes in the new version. It is still a little buggy, i
can handle this for my purpose.
Stellarium now gives the correct value for an integer longitude,
e.g.parameter--longitude= -70 gives the correct position.
But the decimal part is still not correct: e.g. longitude -70.5 gives a
position of -69.5.

Yours

uli

Am Mo., 13. Dez. 2021 um 09:57 Uhr schrieb uli.barkow ***@***.***
